src/components/sidebar-entities/settings-sidebar/index.js

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+
+import { updateSettings } from 'store/actions/settings.actions';
+
+import SettingsSidebar from './settings-sidebar';
+
+const mapStateToProps = state => ({
+  settings: state.settings,
```

Comment:
Same here with defining this in base selectors.

Reply:
Fixed in Irame/sectors-without-number@3e8c22179164e4aa335f4a4e3daa4b3a158d39d8